File: NativeLibraryInterop/native/app/src/main/java/com/example/native_library_wrapper/MainActivity.kt
package com.example.native_library_wrapper

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.wrapper.LottieWrapperView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<LottieWrapperView>(R.id.lottie_wrapper).apply {
            setAnimation(R.raw.bullseye)
            setLoop(true)
            playAnimation()
        }
    }
}

File: App/MainActivity.cs
using Com.Example.Wrapper;

namespace App;

[Activity(Label = "@string/app_name", MainLauncher = true)]
public class MainActivity : Activity
{
    protected override void OnCreate(Bundle? savedInstanceState)
    {
        base.OnCreate(savedInstanceState);

        // Set our view from the "main" layout resource
        SetContentView(Resource.Layout.activity_main);

        var lottieView = FindViewById<LottieWrapperView>(Resource.Id.lottie_wrapper);
        lottieView.SetAnimation(Resource.Raw.bullseye);
        lottieView.SetLoop(true);
        lottieView.PlayAnimation();
    }
}